html/templates/playerdisplaycombatant.html

```diff
@@ -2,8 +2,8 @@
 <div class="name">
     <span data-bind="text: Name, attr: {title: Name }"></span>
 </div>
-<div class="hp">
-    <span data-bind="html: HPDisplay, style: { color: HPColor }" class="current-hp"></span>
+<div class="hp" data-bind="style: { pointerEvents: AllowSuggestions ? 'initial' : 'none' }">
```

Comment:
looks like you're using the pointer-events style to disable both the hover and click behaviors. I think it would be better to disable the hover style here using this logic, and instead check the flag and return early in the body of CombatantSuggestor.Show() on click.